### PR TITLE
Update SiPixelRawToDigi.cc

### PR DESCRIPTION
**Pull request by tvami**
Adding the option for Cabling Map to have a label (default is no label of course)

### CODE REVIEW

File: EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigi.cc [hunk 1]
```diff
@@ -97,6 +98,11 @@ SiPixelRawToDigi::SiPixelRawToDigi( const edm::ParameterSet& conf )
     usePhase1 = config_.getParameter<bool> ("UsePhase1");
     if(usePhase1) edm::LogInfo("SiPixelRawToDigi")  << " Use pilot blade data (FED 40)";
   }
+  //CablingMap could have a label //Tav
+  CablingMapLabel = ""; 
+  if (config_.exists("CablingMapLabel")) {
```
Comment (slava77) [inline EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigi.cc]: this "if" is not necessary after the value is added to the ::fillDescriptions method

**Comment by slava77** [comment]
please also make a PR for 80X

**Comment by tvami** [comment]
@slava77 I did the PR in 8X, do you wish me to correct the if statement of can I leave it like this? Thanks!

**Comment by slava77** [comment]
please correct the "if".
ParameterSet::exists should be used only in exceptional cases, if implementation in fillDescriptions is not possible.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
+1

For #12897 52958a3994a6cf84ba1da968a4aba2c965301977

Adding optional cable map label to SiPixelRawToDigi. There should be no change in monitored quantities.

The code change is satisfactory, and Jenkins [tests](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-12897/10451/summary.html) against baseline CMSSW_7_5_X_2016-01-05-2300 show no significant differences, as expected.

**Comment by davidlange6** [comment]
+1

**Pull request merged**